.travis.yml

```diff
@@ -45,6 +51,7 @@ matrix:
 
   allow_failures:
     - php: hhvm
+    - env: PHPSTAN=1
```

Comment:
@ondrejmirtes https://travis-ci.org/cakephp/cakephp/builds/187472461  so @dereuromark is right.